tests/qiskit_convert_test.py

```diff
@@ -267,17 +269,14 @@ def test_tketpass() -> None:
     assert np.allclose(u1, u2)
 
 
-def test_tketautopass() -> None:
+@pytest.mark.skipif(skip_remote_tests, reason=REASON)
+def test_tketautopass(manila_backend: IBMQBackend) -> None:
     backends = [
         Aer.get_backend("aer_simulator_statevector"),
         Aer.get_backend("aer_simulator"),
         Aer.get_backend("aer_simulator_unitary"),
     ]
-    if not skip_remote_tests:
-        if not IBMQ.active_account():
-            IBMQ.load_account()
-        provider = IBMQ.providers(hub="ibm-q")[0]
-        backends.append(provider.get_backend("ibmq_manila"))
+    backends.append(manila_backend._backend)  # type: ignore
```

Comment:
Does this still need the type ignore with the recent changes?

Reply:
I'll try without